core/src/main/scala/org/apache/spark/scheduler/LiveListenerBus.scala

```diff
@@ -25,14 +25,13 @@ import scala.collection.JavaConverters._
 import scala.collection.mutable
 import scala.reflect.ClassTag
 import scala.util.DynamicVariable
-
```

Comment:
This would fail the scala style check

Reply:
> Under what scenario would this metrics be useful?
This a supplyment for uncompleted work of "droppedEventsCounter" (in LiveListenerBus). 
This metrics could let us know the running condition of bus  at the level of LiveListenerBus, rather than multiple AsyncEventQueue separately only.